apps/files_external/lib/Command/Option.php

```diff
@@ -35,10 +35,10 @@ protected function configure(): void {
 	 */
 	protected function getOption(StorageConfig $mount, $key, OutputInterface $output): void {
 		$value = $mount->getMountOption($key);
-		if (!is_string($value)) { // show bools and objects correctly
+		if (!is_string($value) && json_decode(json_encode($value)) === $value) { // show bools and objects correctly
 			$value = json_encode($value);
```

Comment:
```suggestion
			$value = json_encode($value) ?: (string)$value;
```
And you can revert the change above and the string cast below.